### DISCUSSION TITLE
Error Running LIS for Two Domains

### DISCUSSION THREAD

**Discussion by GGDLW** [NASA-LIS/LISF#1447] [Q&A] [answered]
Hello,

I am trying to get LIS to run for two domains. However, when attempting to do so, I keep getting this error:

`forrtl: severe (151): allocatable array is already allocated`

I am unsure if this error is a result of bad input or a problem with how LIS was installed. I have attached my config file, and my log file. I run this off a batch job from a computing center. The computing center has installed LIS with the following specs.

```
Intel Compiler version 19
OpenMPI version 4     Python 3.9     OpenJPEG 2.4     ESMF 8.2     sZIP 2.1
HDF5 1.12     libPNG 1.6     ECCodes 2.24     HDF4 4.2     zLib 1.2
NetCDF 4.7      LaPack 3.9     Intel-MKL 19     HDF-EOS2 2.20     LibJPEG 2.0
MinPACK 19961126     GDAL 3.4      FortranGIS 2.5      LibTIFF 4.1
```

[lis.config.txt](https://github.com/NASA-LIS/LISF/files/13321361/lis.config.txt)
[lislog_12km.0000.txt](https://github.com/NASA-LIS/LISF/files/13321365/lislog_12km.0000.txt)

**Comment by dmocko** [comment, marked as answer]
Hi @GGDLW 

You might try changing this line in your lis.config file to:

`Number of surface model types:   1`

You correctly have two nests, and correctly have two columns of `"LSM"` for `Surface model types:`.

However, you only have one surface model type for each of your two nests.  LIS is failing because it is looking for a total of 4 `Surface model types`.

If this does not solve your issue, please also attach your ldt.config file to see how you set your domain.

**Reply by GGDLW** [reply]
This solved my issue! Thank you very much for the explanation as well!
